app/models/webauthn_credential.rb

```diff
@@ -0,0 +1,12 @@
+class WebauthnCredential < ApplicationRecord
+  belongs_to :user
+
+  validates :external_id, uniqueness: true, presence: true
+  validates :public_key, presence: true
+  validates :nickname, presence: true
+  validates :sign_count, presence: true, numericality: true
+
+  validate do
+    errors.add(:user, I18n.t("webauthn_credentials.errors.mfa_disabled")) if user.mfa_disabled?
```

Comment:
agree with that validation? I think it makes sense and makes for a better user experience